app/scenes/Activity/ActivityComponent.js

```diff
@@ -83,14 +82,23 @@ const ActivityComponent = ({
         )
         : <View style={{ flex: 1 }} />
       }
-      <ActivityButtons
-        nextLabel={getNextLabel(isLast, isValid, isSkippable)}
-        onPressNext={isValid || isSkippable ? onNext : undefined}
-        prevLabel={getPrevLabel(index === 0, hasPrevPermission)}
-        onPressPrev={onPrev}
-        actionLabel={getActionLabel(answers[index], activity.items[index])}
-        onPressAction={onUndo}
-      />
+      <View style={{
+        backgroundColor: 'white',
+        shadowOpacity: 0.75,
+        shadowRadius: 5,
+        shadowColor: 'lightgray',
+        shadowOffset: { height: 0, width: 0 },
```

Comment:
Android doesn't respect these shadow props. You can fake it with the `elevation` style property, which is a combination of z-index and shadow.

Reply:
By which I mean, if you want a drop shadow on Android as well as iOS, you could add another line like `elevation: 2`